utils/xhrModule.js

```diff
@@ -90,9 +89,6 @@ class XHRQueue {
               )
             );
           } else {
-            // logger.debug(
```

Comment:
Add some comments to identify this is the successful case

Reply:
It didn't make sense to log the success case. Hence, I removed even the previously commented-out log statement.

Reply:
Please add a comment to understand after the request is successful it will come to this block.

Reply:
Why do we need logs for a success case? It makes more sense for errors, right?